examples/nodejs/lambda-examples/topics-microservice/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "topics-microservice",
+  "version": "0.1.0",
+  "author": "NoSQLKnowHow",
+  "description": "Using the AWS SDK for JavaScript V3 to build and deploy Lambda functions on API Gateway.",
+  "license": "Apache 2.0",
+  "bin": {
+    "topics-microservice": "bin/topics-microservice.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
```

Comment:
this is what allows you to do `npm run cdk`, which prevents users from needing to install `cdk`, and ensures that they are using the right version.